### ISSUE TITLE
update singularity variable

### ISSUE THREAD

**Issue by pappewaio** [BioPsyk/cleansumstats#389]
![Screenshot 2023-06-21 at 12 19 40](https://github.com/BioPsyk/cleansumstats/assets/1268138/dd548868-96ad-4676-86a5-04b95b33f2cb)

**Comment by pappewaio** [comment]
```
WARNING: DEPRECATED USAGE: Environment variable SINGULARITY_HOME will not be supported in the future, use APPTAINER_HOME instead
WARNING: /etc/singularity/ exists, migration to apptainer by system administrator is not complete
```

**Comment by pappewaio** [comment]
Updating from `SINGULARITY_HOME` to `APPTAINER_HOME` in cleansumstats.sh gives this error:
```
[jesgaaopen@fe-open-01 cleansumstats]$ ./cleansumstats.sh -e
CAPSULE EXCEPTION: Capsule not extracted. while processing attribute App-Class-Path: [/usr/bin/nextflow, [nextflow-21.12.1-edge.jar, ivy-2.3.0.jar, nf-httpfs-21.12.1-edge.jar, nf-ignite-1.2.3.jar, nf-commons-21.12.1-edge.jar, groovy-nio-3.0.9.jar, groovy-templates-3.0.9.jar, groovy-xml-3.0.9.jar, groovy-json-3.0.9.jar, groovy-3.0.9.jar, jcl-over-slf4j-1.7.32.jar, jul-to-slf4j-1.7.32.jar, log4j-over-slf4j-1.7.32.jar, logback-classic-1.2.9.jar, logback-core-1.2.9.jar, gpars-1.2.1.jar, grengine-1.3.0.jar, commons-lang-2.6.jar, pf4j-update-2.3.0.jar, commons-codec-1.10.jar, jcommander-1.35.jar, kryo-2.24.0.jar, leveldb-0.12.jar, org.eclipse.jgit-5.2.1.201812262042-r.jar, mail-1.4.7.jar, activation-1.1.1.jar, jackson-dataformat-yaml-2.10.1.jar, snakeyaml-1.28.jar, jsoup-1.14.3.jar, jline-2.9.jar, pf4j-3.4.1.jar, failsafe-3.1.0.jar, json-schema-validator-1.0.47.jar, ignite-core-2.4.0.jar, ignite-slf4j-2.4.0.jar, guava-24.1.1-jre.jar, slf4j-api-1.7.32.jar, multiverse-core-0.7.0.jar, jsr166y-1.7.0.jar, objenesis-2.1.jar, leveldb-api-0.12.jar, jsch-0.1.54.jar, jzlib-1.1.1.jar, JavaEWAH-1.1.6.jar, java-semver-0.9.0.jar, jackson-databind-2.10.4.jar, commons-lang3-3.7.jar, jackson-core-2.10.4.jar, cache-api-1.0.0.jar, annotations-13.0.jar, ignite-shmem-1.0.0.jar, jsr305-1.3.9.jar, checker-compat-qual-2.0.0.jar, error_prone_annotations-2.1.3.jar, j2objc-annotations-1.1.jar, animal-sniffer-annotations-1.14.jar, gson-2.2.4.jar, jackson-annotations-2.10.4.jar, nf-amazon-1.4.0.jar, aws-java-sdk-s3-1.12.129.jar, aws-java-sdk-ec2-1.12.129.jar, aws-java-sdk-batch-1.12.129.jar, aws-java-sdk-iam-1.12.129.jar, aws-java-sdk-ecs-1.12.129.jar, aws-java-sdk-logs-1.12.129.jar, tika-core-1.5.jar, aws-java-sdk-kms-1.12.129.jar, aws-java-sdk-core-1.12.129.jar, httpclient-4.5.13.jar, commons-codec-1.15.jar, jmespath-java-1.12.129.jar, jackson-dataformat-cbor-2.12.3.jar, jackson-databind-2.12.3.jar, jackson-core-2.12.3.jar, jackson-annotations-2.12.3.jar, jackson-dataformat-yaml-2.12.3.jar, ion-java-1.0.2.jar, joda-time-2.8.1.jar, httpcore-4.4.13.jar, nf-google-1.1.4.jar, google-api-services-lifesciences-v2beta-rev20210527-1.31.5.jar, google-cloud-nio-0.121.2.jar, google-cloud-storage-1.109.1.jar, google-api-services-storage-v1-rev20200430-1.30.9.jar, google-api-client-1.31.5.jar, google-cloud-core-1.93.6.jar, google-oauth-client-1.31.5.jar, google-http-client-gson-1.39.2.jar, google-http-client-apache-v2-1.39.2.jar, gax-1.57.0.jar, google-auth-library-oauth2-http-0.20.0.jar, google-http-client-jackson2-1.35.0.jar, google-http-client-1.39.2.jar, commons-codec-1.11.jar, proto-google-iam-v1-0.13.0.jar, api-common-1.9.2.jar, protobuf-java-util-3.12.2.jar, opencensus-contrib-http-util-0.28.0.jar, guava-30.1.1-android.jar, jackson-core-2.11.0.jar, javax.inject-1.jar, auto-value-annotations-1.7.3.jar, jsr305-3.0.2.jar, google-auth-library-credentials-0.20.0.jar, gson-2.8.6.jar, httpcore-4.4.14.jar, failureaccess-1.0.1.jar, listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar, checker-compat-qual-2.5.5.jar, error_prone_annotations-2.5.1.jar, j2objc-annotations-1.3.jar, proto-google-common-protos-1.18.0.jar, google-cloud-core-http-1.93.6.jar, google-http-client-appengine-1.35.0.jar, gax-httpjson-0.74.0.jar, javax.annotation-api-1.3.2.jar, opencensus-api-0.28.0.jar, grpc-context-1.30.0.jar, protobuf-java-3.12.2.jar, threetenbp-1.4.4.jar, nf-tower-1.3.0.jar], /cleansumstats/ibp-pipeline-lib.jar] (for stack trace, run with -Dcapsule.log=verbose)
Unable to initialize nextflow environment
cleansumstats.sh reached the end: Wed Jul  5 23:12:05 CEST 2023
>> Cleaning up (disable with -l) 
>> Removing /faststorage/project/ibp_pipeline_cleansumstats/cleansumstats/out_test/.nextflow
rm: cannot remove ‘/faststorage/project/ibp_pipeline_cleansumstats/cleansumstats/out_test/.nextflow’: No such file or directory
>> Done

```

**Comment by pappewaio** [comment]
solved by PR #399
